tests/playwright-test/playwright-test-fixtures.ts

```diff
@@ -367,8 +367,8 @@ export const test = base
 const TSCONFIG = {
   'compilerOptions': {
     'target': 'ESNext',
-    'moduleResolution': 'node',
-    'module': 'commonjs',
+    'moduleResolution': 'bundler',
```

Comment:
why did this change?

Reply:
This updates our default tsc config in tests to align with what we aim to support, particularly "bundler". Let me land it separately.